Snakemake/workflow/profile/cluster.json

```diff
@@ -13,8 +13,8 @@
  },
  "phase":
   {
-    "mem": "150G",
-    "threads": "20",
+    "mem": "350G",
+    "threads": "25",
```

Comment:
Here, you need to change to 25G

Reply:
Is this 25, the number of threads or RAM per thread?